kyaml/kio/filters/container.go

```diff
@@ -25,14 +25,16 @@ import (
 // The full set of environment variables from the parent process
 // are passed to the container.
 type ContainerFilter struct {
-	mountPath string
 
 	// Image is the container image to use to create a container.
 	Image string `yaml:"image,omitempty"`
 
 	// Network is the container network to use.
 	Network string `yaml:"network,omitempty"`
 
+	// List of storage options that container will have mounted.
```

Comment:
nit: go comment style is `FIELD_NAME description` -- e.g. `StorageMounts is a list of ...`